firmware/hw_layer/sensors/accelerometer.cpp

```diff
@@ -150,33 +224,70 @@ void initAccelerometer() {
 	}
 
 	/* Commented until we have configureHellenMegaAccCS2Pin() */
-	#if 0
+	#ifndef HW_HELLEN
```

Comment:
I do not get it. how is HW_HELLEN relevant here?

Reply:
> I do not get it. how is HW_HELLEN relevant here?

Helen shares same SPI for SD and Accel. Hellen has `configureHellenMegaAccCS2Pin() `that does:
```
	    cs2pin.initPin("mm-CS2", Gpio::H_SPI1_CS2);
	    cs2pin.setValue(1);
```
And does not release this `cs2pin`. Attempt to init this pin in accelerometer.cpp will fail on Hellen boards.